make-hash.js

```diff
@@ -1,4 +1,7 @@
 'use strict'
+if (typeof Buffer === 'undefined') {
+  global.Buffer = require('buffer').Buffer
```

Comment:
Indeed, I've done it by importing the shim.js file as explained [here](https://github.com/39otrebla/react-native-bitcoinjs-lib). And the global patch works fine with all modules but this. I am not sure why, the only thing I came up with is the 'use-strict' directive. But I couldn't find an actual reason why 'use-strict' should "break" the patch.